[2023-10-07 12:25:07,174] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.update_f_sales scheduled__2023-10-01T12:19:33.489881+00:00 [queued]>
[2023-10-07 12:25:07,182] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.update_f_sales scheduled__2023-10-01T12:19:33.489881+00:00 [queued]>
[2023-10-07 12:25:07,183] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-07 12:25:07,184] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-07 12:25:07,185] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-07 12:25:07,197] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): update_f_sales> on 2023-10-01 12:19:33.489881+00:00
[2023-10-07 12:25:07,201] {standard_task_runner.py:52} INFO - Started process 1904 to run task
[2023-10-07 12:25:07,203] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'sales_mart', 'update_f_sales', 'scheduled__2023-10-01T12:19:33.489881+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/sprint3.py', '--cfg-path', '/tmp/tmpa_1ov6em', '--error-file', '/tmp/tmpo0dxj4n8']
[2023-10-07 12:25:07,204] {standard_task_runner.py:80} INFO - Job 50: Subtask update_f_sales
[2023-10-07 12:25:07,245] {task_command.py:369} INFO - Running <TaskInstance: sales_mart.update_f_sales scheduled__2023-10-01T12:19:33.489881+00:00 [running]> on host d964b2b568e5
[2023-10-07 12:25:07,287] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=student@example.com
AIRFLOW_CTX_DAG_OWNER=student
AIRFLOW_CTX_DAG_ID=sales_mart
AIRFLOW_CTX_TASK_ID=update_f_sales
AIRFLOW_CTX_EXECUTION_DATE=2023-10-01T12:19:33.489881+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-01T12:19:33.489881+00:00
[2023-10-07 12:25:07,294] {base.py:68} INFO - Using connection ID 'postgresql_de' for task execution.
[2023-10-07 12:25:07,298] {sql.py:315} INFO - Running statement: insert into mart.f_sales (date_id, item_id, customer_id, city_id, quantity, payment_amount,"status")
select 
    dc.date_id
    ,uol.item_id
    ,uol.customer_id
    ,uol.city_id
    ,uol.quantity
    ,case 
        when uol."status" = 'refunded'
        then -uol.payment_amount 
        else uol.payment_amount 
    end as payment_amount
    ,uol."status"
from staging.user_order_log uol
left join mart.d_calendar as dc on uol.date_time::Date = dc.date_actual
where uol.date_time::Date = '2023-10-01';, parameters: {'date': ('{{ ds }}',)}
[2023-10-07 12:25:07,301] {sql.py:324} INFO - Rows affected: 116
[2023-10-07 12:25:07,310] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=sales_mart, task_id=update_f_sales, execution_date=20231001T121933, start_date=20231007T122507, end_date=20231007T122507
[2023-10-07 12:25:07,336] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-07 12:25:07,360] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
